pkg/tarmak/provider/amazon/amazon.go

```diff
@@ -521,20 +516,35 @@ func (a *Amazon) VerifyInstanceTypes(instancePools []interfaces.InstancePool) er
 	for _, instance := range instancePools {
 		instanceType, err := a.InstanceType(instance.Config().Size)
 		if err != nil {
-			return err
+			result = multierror.Append(result, err)
+			continue
 		}
 
 		if err := a.verifyInstanceType(instanceType, instance.Zones(), svc); err != nil {
 			result = multierror.Append(result, err)
 		}
+
+		if instance.Name() == "master" {
```

Comment:
I think we also should do a check for vault and etcd. I don't think we should run these on burstable (t) types.

Reply:
done